src/bp/core/services/cms/util.ts

```diff
@@ -23,8 +24,12 @@ export class SafeCodeSandbox {
     this.tmpPath = this.tmpDir.name
 
     for (const file of files) {
-      const filePath = path.resolve(path.join(this.tmpDir.name, file.relativePath))
+      const pt = path.resolve(path.join(this.tmpDir.name, file['folder']))
+      fse.mkdirpSync(pt)
+      // TODO: use the correct module path
```

Comment:
Can you make this TODO more explicit? i.e. explain what the problem is so that everybody will understand it